java/simulator/src/main/java/com/hazelcast/simulator/probes/impl/HdrLatencyProbe.java

```diff
@@ -28,6 +30,8 @@ public class HdrLatencyProbe implements LatencyProbe {
     // we want to track up to 24-hour.
     static final long HIGHEST_TRACKABLE_VALUE_NANOS = DAYS.toNanos(1);
 
+    private final AtomicLong negativeCount = new AtomicLong();
```

Comment:
So far I have not had a single run where this happened (and I have been using Simulator for a long long time). So since it should happen frequently, this is the simplest solution.